src/test/java/it/pagopa/pagopa/apiconfig/service/BrokersServiceTest.java

```diff
@@ -54,8 +55,37 @@ void getBroker() throws IOException, JSONException {
         BrokerDetails result = brokersService.getBroker("1234");
         String actual = TestUtil.toJson(result);
         String expected = TestUtil.readJsonFromFile("response/get_broker_ok1.json");
-        System.out.println(actual);
         JSONAssert.assertEquals(expected, actual, JSONCompareMode.STRICT);
     }
 
+    @Test
+    void createBroker() throws IOException, JSONException {
+        when(intermediariPaRepository.findByIdIntermediarioPa("1234")).thenReturn(Optional.empty());
+        when(intermediariPaRepository.save(any(IntermediariPa.class))).thenReturn(getMockIntermediariePa());
+
+        BrokerDetails result = brokersService.createBroker(getMockBrokerDetails());
+        String actual = TestUtil.toJson(result);
+        String expected = TestUtil.readJsonFromFile("response/create_broker_ok.json");
```

Comment:
it would be useful not to have only happy case

Reply:
You are right